packages/frontend/src/hooks/contracts/useCrabStrategy.ts

```diff
@@ -0,0 +1,193 @@
+import { useWallet } from '@context/wallet'
```

Comment:
@alexisgauba  I moved this file to a context in fe/strategy. If you made any changes here move it to the context file and remove this file

Reply:
already added to context! lemme remove the file